.github/workflows/tests.yml

```diff
@@ -35,6 +35,8 @@ jobs:
                   -DSINGULARITY_BUILD_EXAMPLES=ON \
                   -DSINGULARITY_BUILD_PYTHON=ON \
                   -DSINGULARITY_TEST_SESAME=OFF \
+                  -DSINGULARITY_USE_HELMHOLTZ=ON \
```

Comment:
@Yurlungur Do we want to have an option for this? It doesn't change anything in the configuration (e.g. require some other dependency or disable some other feature). If it's just a new EoS in a header file, treat it like all the others. The test can be put into the standard test targets.

I suppose that it could be the a case that since the data directory likely needs to be part of the install, but that would probably be standard anyway. Your call.

Reply:
I think the main benefit is it prevents it from being in the variant by default. That said, we could just not include it in the variant, and remove the build option.

Reply:
In the limit of infinite EOSes in `singularity-eos`, is the solution to infinite compile times due to variant to give each EOS, not just Helmholtz, a similar enable/disable flag?

Reply:
I think the long term plan is not to provide a single variant, but instead have a few predefined variants for different use-cases. And if those don't meet needs, then the downstream code builds their own. TO enable this, I need to refactor `EOSBuilder`, which is on my TODO list. This enable/disable flag is a short-term hack that gets us short compile times until I can get that cleaned up.

Reply:
Naively I would think it would be easier to let the downstream code request the EOSes they want through compile time flags like this rather than build their own variant which can be time consuming to work out. Is it difficult for `singularity-eos` to provide a single mega-variant that is pruned down at compile time through options like this? Such a mega-variant could presumably also be pre-pruned down into the predefined variants you have in mind,

Reply:
Yeah I'm open to suggestions here, including the mega variant with pruning options.

My main concern about that is that the combinatoric list of EOS's and variants might be quite large and complex. Letting the downstream C++ codes build their own variants moves some of that burden away from singularity-eos. 

That said, a few predefined variants probably makes sense for common use cases... e.g., I want all the astro-relevant EOS's or all the terrestrial-relevant EOS's. That kind of thing.

Reply:
If the issue is compile times, I'd also like to know exactly where the compilation is getting stuck. @dholladay00 today just told me that `PTofRE()` might be a prime target for example.

Reply:
@jhp-lanl `PTofRE` is on my list to remove entirely. The improved compile times should now be available in the `main` branch presently.

Reply:
I am open to both encouraging downstream codes to use the variadic utils to create their own variants as well as enabling the main variant to have configuration-time toggle-able options. I'd love to hear of folks' thoughts on how to best manage that. Is there a way to get cmake to do some of that code-gen magic for us @mauneyc-LANL ? Then there's the question of if we expose this at the cmake level do we also expose it at the spack level?

Long term, I think the best solution would be to JIT the variant based on requested models in a simulation, but that is a distant dream at this point. 

This is probably best suited for a broader discussion and not in this PR.